pages/fundamentals/indexes.mdx

```diff
@@ -187,6 +187,42 @@ Named parameters are not supported for edge-type property indexes.
 
 </Callout>
 
+### Point index
+
+To run queries such as:
+```cypher
+WITH point({x:1, y:1}) AS p
+MATCH (node:Label) WHERE point.distance(p, node.property) < 1
+RETURN node;
+```
+in a performant way, feel free to create a new point index as follows:
+```cypher
+CREATE POINT INDEX ON :Label(property);
+```
+
+<Callout type="info">
+
+In Memgraph version 2.21 to utilize the point index in queries using point.distance the comparison point
+needs to exist as its own variable
+For example instead of:
```

Comment:
```suggestion
From Memgraph 2.21, to utilize the point index in queries using the point.distance function, the comparison point
needs to exist as a variable first. For example, instead of:
```